src/security/issues/sync.py

```
@@ -359,36 +371,25 @@ def _handle_new_child_issue(
+            sync.stats.children_linked += 1
+        else:
             logging.info(
-                "DRY-RUN: add sub-issue link parent=#%d child=(new) alert=%d",
-                parent_issue.number,
+                DRY_RUN_PREFIX + "Would create child issue for alert %d (rule: %s, severity: %s)",
                 ctx.alert_number,
+                ctx.rule_id,
+                ctx.severity,
             )
+            if ctx.rule_id:
+                logging.debug("No parent issue yet for rule_id=%s – link will happen on next sync", ctx.rule_id)
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)